stm/asyncTaskQueue.ml

```diff
@@ -123,7 +123,7 @@ module Make(T : Task) = struct
                  "-worker-id"; name;
                  "-async-proofs-worker-priority";
                    Flags.string_of_priority !Flags.async_proofs_worker_priority]
-        | ("-ideslave"|"-emacs"|"-emacs-U"|"-batch")::tl -> set_slave_opt tl
```

Comment:
AFAIR `-emacs` was using characters >= 128 for the emacs communication (e.g. 249) and this was broking transferring utf8. The option `-emacs-U` was introduced to support utf8 (f8d64ae9). After a transition period, `-emacs` started itself to call the code of `-emacs-U` and `-emacs-U` be deprecated (77f1db2).